bootstrap_conan_utils.py

```diff
@@ -0,0 +1,565 @@
+#!/usr/bin/env python
+
+from dataclasses import dataclass
+
+import json
+import os
+import platform
+import requests
```

Comment:
I think we need to update the `README` to indicate that we need to install the `requirements.txt` in a venv or something before running the `bootstrap.py` script.

Reply:
I tried to bootstrap and ran into this:

```
/Users/callendorph/src/prs/slm/.slm/deps/file_system_utils_static/stanza.proj:23.6: Cannot resolve variable STANZA_CONFIG.
/Users/callendorph/src/prs/slm/.slm/deps/file_system_utils_static/stanza.proj:27.6: Cannot resolve variable STANZA_CONFIG.
/Users/callendorph/src/prs/slm/.slm/deps/file_system_utils_static/stanza.proj:31.6: Cannot resolve variable STANZA_CONFIG.
/Users/callendorph/src/prs/slm/.slm/deps/json_static/stanza.proj:23.6: Cannot resolve variable STANZA_CONFIG.
/Users/callendorph/src/prs/slm/.slm/deps/json_static/stanza.proj:27.6: Cannot resolve variable STANZA_CONFIG.
/Users/callendorph/src/prs/slm/.slm/deps/json_static/stanza.proj:31.6: Cannot resolve variable STANZA_CONFIG.
Bootstrap failed trying to run `stanza build`
```

Prob needs docs too.

Reply:
There's actually a ticket in the slm project Linear for adding venv and requirements.txt bootstrapping.  I'll add it to the README for now.

Reply:
Done.  The STANZA_CONFIG bit is messy for the user, but I'm not sure how to make it better at the moment.  I'll create a ticket to streamline.